CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ add_compile_options(-Wall)
 find_package(PkgConfig)
 
 pkg_search_module(LIBLDNS REQUIRED libldns ldns)
-pkg_check_modules(LIBUV REQUIRED libuv)
+pkg_check_modules(LIBUV REQUIRED libuv>=1.23)
```

Comment:
If we're going to check the version, we might want to break on libuv 2 or more, since libuv uses semantic versioning:
http://docs.libuv.org/en/v1.x/version.html
That means that a bump to 2 or more is potentially incompatible.

Reply:
I can't figure out a way to make pkg_check_modules have both a lower and an upper limit ...

Reply:
I think it's unusual to check for max version. This should be good.